ploneorg/theme/theme/plone-open-garden.html

```diff
@@ -2426,10 +2387,10 @@ <h2>How do I get there?</h2>
   <div class="container">
   <div class="span12">
       <h2 class="short">Register now</h2>
-      <p>If you’ve decided to come, fill out the <a href="/plog-registration-form">PLOG
+      <p>If you’ve decided to come, fill out the <a href="/plog-registration-form-2020">PLOG
           registration form</a>.</p>
-      <p>Let us know if you're coming! The deadline is February 28, 2019.</p>
-      <a class="btn btn-flat btn-primary" href="/plog-registration-form">
+      <p>Let us know if you're coming! The deadline is February 28, 2020.</p>
+      <a class="btn btn-flat btn-primary" href="/plog-registration-form-2020">
```

Comment:
Same idea... keep the URLs constant from year to year and mention only generic dates but link to the specific anchor on the registration form that shows the actual deadline.

Reply:
Same questions as above.

Reply:
up to you

Reply:
Added exhortation to remember the deadline.